### PR TITLE
Adds dunnjacoba.json

### PR DESCRIPTION
**Pull request by dunnjacoba**
## Changes proposed
Adds dunnjacoba.json

## Check List (Check all the applicable boxes) 

- [X] My code follows the code style of this project.
- [ ] My change requires changes to the documentation.
- [ ] I have updated the documentation accordingly.
- [X] All new and existing tests passed.
- [X] This PR does not contain plagiarized content.
- [X] The title of my pull request is a short description of the requested changes.

## Note to reviewers

Thank you all for being so inclusive of even beginners like myself! I am so glad to have found this community.

<a href="https://gitpod.io/#https://github.com/EddieHubCommunity/LinkFree/pull/1607"></a>

### CODE REVIEW

**Review by kumarsonsoff3 — CHANGES_REQUESTED** [review]
Please revert the changes from the `package-lock.json` file. Let us know if you need any help with this! 🛠

__
[sema-logo](https://semasoftware.com/gh) &nbsp;**Summary:** :hammer_and_wrench: This code needs a fix&nbsp; | &nbsp;**Tags:** Inelegant

**Comment by dunnjacoba** [comment]
> Please revert the changes from the `package-lock.json` file. Let us know if you need any help with this! 🛠
> 
> __ [sema-logo](https://semasoftware.com/gh) [ sema-logo ](https://semasoftware.com/gh) [ ](https://semasoftware.com/gh)  **Summary:** 🛠️ This code needs a fix  |  **Tags:** Inelegant

If possible, please give instruction on how to revert the package-lock,json, the changes were not made intentionally, I believe it happened when I ran npm install

File: package-lock.json [hunk 1]
```diff
@@ -6,7 +6,7 @@
   "packages": {
     "": {
       "name": "linkfree",
-      "version": "0.70.1",
+      "version": "0.70.17",
```
Comment (kumarsonsoff3) [inline package-lock.json:9]: Package version back to the one in the base branch! 

```suggestion
      "version": "0.70.1",
```

**Review by kumarsonsoff3 — CHANGES_REQUESTED** [review]
Yea @dunnjacoba, that's true, these changes happen because your IDE is using the latest version of these packages! But you should have only committed your file! 

No worries ~~

Just makes these 2 changes! 
- Line no `9`, version back to initial as suggested
- Remove the `empty endline` from the `package.lock.json` file

File: package-lock.json (hunk 1 above)
Comment (dunnjacoba) [reply package-lock.json]: Corrected, after a couple of tries. Sorry that took me so long to get right, but now it registers as the addition of the profile json as the only difference from the main branch.

File: public/data/dunnjacoba.json [hunk 2]
```diff
@@ -0,0 +1,41 @@
+{
+  "name": "Jacob Dunn",
+  "type": "personal",
+  "bio": "Freelance Dev, Open source beginner",
+  "avatar": "https://github.com/dunnjacoba.png",
+  "links": [
+    {
+      "name": "Check out my green boxes on Github!",
```
Comment (Panquesito7) [inline public/data/dunnjacoba.json:8]: I recommend capitalizing the `h` here. What do you think?
```suggestion
      "name": "Check out my green boxes on GitHub!",
```


__
[sema-logo](https://semasoftware.com/gh) &nbsp;**Summary:** :question: I have a question

**Review by kumarsonsoff3 — APPROVED** [review]
LGTM 👍 
Thanks for contributing! 🚀 

__
[sema-logo](https://semasoftware.com/gh) &nbsp;**Summary:** :ok_hand: This code looks good&nbsp; | &nbsp;**Tags:** Elegant

**Review by EmmaDawsonDev — APPROVED** [review]

**Pull request merged**